src/tests/tracing/eventpipe/applystartuphook/ApplyStartupHookValidation.cs

```diff
@@ -82,10 +168,21 @@ private static IpcMessage CreateApplyStartupHookMessage(string startupHookPath)
             return new IpcMessage(0x04, 0x07, serializedConfiguration);
         }
 
+        private static void CheckResponse(IpcMessage response, ref bool fSuccess)
+        {
+            bool success = response.Header.CommandId == (byte)0; // DiagnosticsServerResponseId.OK
+            string status = success ? "success" : "failed";
+            Logger.logger.Log($"Response status: {status}");
```

Comment:
Is it better to just log the command ID here, or log it additionally. Also, any reason for ref bool instead of just letting it be the return of the function? Let the aggregation be responsibility of the caller.